packages/spacecat-shared-rum-api-client/src/functions/form-vitals.js

```diff
@@ -16,6 +16,8 @@ import { generateKey, DELIMITER, loadBundles } from '../utils.js';
 
 const FORM_SOURCE = ['.form', '.marketo', '.marketo-form'];
 const METRICS = ['formview', 'formengagement', 'formsubmit', 'formbuttonclick'];
+const FILTER_EVENTS_CHECKPOINT = ['click', 'viewblock', 'formsubmit', 'fill'];
```

Comment:
This already happens via exporting a checkpoint list in this file.
We should add the fill checkpoint in that list instead